repose-aggregator/external/os-auth-schemas/src/main/resources/META-INF/schema/openstack/OS-KSCATALOG.xsd

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="../xslt/schema.xslt"?>
+<?xml-stylesheet type="text/xsl" href="schema.xsl"?>
```

Comment:
I'm totally floored that with this many type and use changes this did not cause any ripple effects.
Do we just not use these portions of the schema?